tests/test_c2pa.py

```diff
@@ -1,5 +1,5 @@
 import os
-import pytest
+import unittest
```

Comment:
I'd like to use only one unit test framework. `unittest` is build-in Python, and acts up less often due to context switches.